app/scripts/views/customizer/CustomizerAppView.js

```diff
@@ -22,13 +22,30 @@ define(['backbone', 'CustomizerViewerView', 'CustomizerHeaderView', 'CustomizerW
 
     render: _.once(function() {
 
-      (new CustomizerViewer()).render();
+      this.threeViewer = new ThreeViewer({container: document.getElementById("customizer-viewer")});
+      this.listenTo(this.model, 'geometry-data-received:event', this.updateNodeGeometry);
+      this.listenTo(this.model.get('workspaces'), 'geometryUpdated', this.updateNodeGeometry);
+
       (new CustomizerHeader({model: this.model.getCurrentWorkspace() })).render();
       (new CustomizerWorkspaceView({model: this.model.getCurrentWorkspace() })).render();
 
       return this;
 
-    })
+    }),
+
+    updateNodeGeometry: function(e){
```

Comment:
CustomizerAppView and AppView have same methods now, but I think It's to early create a base class for them, because there are just a couple of same lines.